hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieClusteringConfig.java

```diff
@@ -48,6 +48,8 @@
         + "which optimizes the storage layout for better query performance by sorting and sizing data files.")
 public class HoodieClusteringConfig extends HoodieConfig {
 
+  public static final boolean DEFAULT_PRESERVE_COMMIT_METADATA = true;
```

Comment:
We don't need the variable and we can assume that the default behavior reflects this.  Similar for the compaction.

Reply:
These are removed.